### ISSUE TITLE
The plugin fails to update if this action is invoked from the "View Details" page

### ISSUE THREAD

**Issue by adriatic** [terrylinooo/githuber-md#67]
**Any issue will be ignored if you don't provide the following detailed information.**

**Describe the bug**

Most often the Wordpress user finds about the new version of the plugin when looking at the list of installed plugins. At this point a typical user would click on the **`View details`** link (see the image below)

![image](https://user-images.githubusercontent.com/2712405/57536244-c055d680-7311-11e9-9d4f-886a0a1ecee4.png)

This click brings up the popup window showing the description of this update. The bottom right corner contains the **`Install Update`** button. The bug being reported here is that this **`Install Update`** button does not install the new version of the plugin (it seems as if it does not do anything)

**Server environment**

 - WordPress version [ e.g. 5.2 ]
 - WP Githuber MD plugin version [ e.g. 1.11.0 ]
 - PHP version [ e.g. 7.2.13]

**User environment**

 - OS: [ Windows 10]
 - Browser [ Chrome, Firebox ]

**Comment by terrylinooo** [comment]
The screenshot shows that the version has been the lastest. Nothing will happen when you click the Update button.

**Comment by adriatic** [comment]
Sorry for the incomplete explanation of this issue: my problem report is correct, I just did not explain all the steps. I saw the message that your plugin is not the newest (that there is a newer version) - and without much thinking I clicked on the update button on the big popup window. Seeing that this click did not result with any action, I dismissed the popup window and clicked on the "update plugin" link on the "plugins window itself". This click resulted with the expected behavior - the plugin got updated to the current version (see below):

![image](https://user-images.githubusercontent.com/2712405/57557222-50fad980-7347-11e9-88f8-2fdee760b1fb.png)

Let me know if this additional explanation helps
